docs/contributor_guide/developer_call_notes/2020_10_08.rst

```diff
@@ -0,0 +1,55 @@
+Developer Call 2020-10-08
+-------------------------
+
+Scribbler: `Richard Edgar <https://github.com/riedgar-ms>`_
+
+Attendees:
+
+- `Miro Dudik <https://github.com/MiroDudik>`_
+- `Richard Edgar <https://github.com/riedgar-ms>`_
+- `Adrin Jalali <https://github.com/adrinjalali>`_
+- `Diondra Peck <https://github.com/diondrapeck>`_
+- Alex Quach
+- `Kevin Robinson <https://github.com/kevinrobinson>`_
+- `Mehrnoosh Sameki <https://github.com/mesameki>`_
+- `Hanna Wallach <https://www.microsoft.com/en-us/research/people/wallach/>`_
+- `Hilde Weerts <https://github.com/hildeweerts>`_
+
+Agenda:
+
+- Requirements for next release
+    - Still need new metrics API
+    - Need to announce deprecation of ``FairnessDashboard``
+    - Need to make sure Quickstart will actually work
+      (short term fix: put note about running `pip install` against GitHub at the top)
+    - Not clear what ``ThresholdOptimizer`` doc fixes Roman had in mind
```

Comment:
FYI I'm making those on the side right now. Essentially there is no user guide right now, and it has several quirks that I'd rather write down. They've come up in at least a handful conversations and clearly documentation could solve that.